src/drivers/imx/interrupt.c

```diff
@@ -3,26 +3,415 @@
 // Copyright 2019 NXP
 //
 // Author: Daniel Baluta <daniel.baluta@nxp.com>
+// Author: Paul Olaru <paul.olaru@nxp.com>
 
 #include <sof/drivers/interrupt.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/io.h>
+#include <sof/common.h>
```

Comment:
Alphabetical order, so this should be the first include.

Reply:
Alright